tests/webview/ui/test_content.py

```diff
@@ -924,3 +925,24 @@ def test_books_containing_list_is_on_left_of_page(webview_base_url, selenium, pa
     # THEN check if the books list exists and on the left
     assert content.books_containing.book_list
     assert content.location['x'] < window_width / 2
+
+
+@markers.webview
+@markers.test_case('C195056')
+@markers.nondestructive
+@markers.parametrize('page_id', ['4fGVMb7P@1'])
+def test_button_open_with_certain_window_size(webview_base_url, selenium, page_id):
+    # GIVEN the webview base url, page_id, and the Selenium driver
+
+    # WHEN we visit that page of the chapter and we have a list of books containing the page
+    content = ContentPage(selenium, webview_base_url, id=page_id).open()
+
+    # THEN if window width >= 640, button should be open
+    window_width = content.get_window_size('width')
+    if window_width >= 640:
```

Comment:
Let's not hardcode things. you can pass in the window dimensions via parametrize.

https://github.com/openstax/cnx-automation/blob/master/tests/webview/ui/test_home.py#L19